### DISCUSSION TITLE
Combien de temps pour obtenir les premières valeurs ?

### DISCUSSION THREAD

**Discussion by Coroebus** [ssenart/PyGazpar#52] [General]
Hello,

j'ai tout paramétré, j'ai bien toutes les entités mais elles n'ont pas de valeurs.
Je n'ai pas relevé d'erreur dans les logs.

Peut-être suis-je trop pressé ;)

Nico

**Comment by ssenart** [comment]
Hello,

Et lorsque tu télécharges le relevé quotidien sur le site de GrDF, tu as bien des valeurs tous les jours ?
